src/NServiceBus.AcceptanceTests/Encryption/EndpointTemplates/DefaultServer.cs

```diff
@@ -42,7 +42,8 @@ public BusConfiguration GetConfiguration(RunDescriptor runDescriptor, EndpointCo
             builder.CustomConfigurationSource(configSource);
             builder.EnableInstallers();
 
-            builder.DisableFeature<TimeoutManager>();
+            // TimeoutManager is currently required by Sagas
+            builder.EnableFeature<TimeoutManager>();
```

Comment:
Why was it explicitly disabled before? Are there any performance considerations here (like test run time)?